prediction_market_agent_tooling/markets/data_models.py

```
@@ -100,6 +103,36 @@ def to_boolean_outcome(value: str | bool) -> bool:
+
+    @property
+    def p_up(self) -> Probability:
+        if self.scalar_value > self.upperBound:
+            return Probability(1)
+        elif self.scalar_value < self.lowerBound:
+            return Probability(0)
+        else:
+            return Probability(
+                (self.scalar_value - self.lowerBound)
+                / (self.upperBound - self.lowerBound)
+            )
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)